klaviyo/campaigns.py

```diff
@@ -0,0 +1,48 @@
+from .api_helper import KlaviyoAPI
+
+
+class Campaigns(KlaviyoAPI):
+    CAMPAIGNS = 'campaigns'
+    CAMPAIGN = 'campaign'
+    RECIPIENTS = 'recipients'
+
+    def get_campaigns(self, page: str = None, count: str = None):
```

Comment:
Can you please remove the type annotations for the time being? This code should still be backwards compatible with python 2 for now.

Reply:
done.